dftimewolf/lib/exporters/timesketch.py

```diff
@@ -53,10 +53,13 @@ def __init__(
   # pylint: disable=arguments-differ
   def SetUp(
       self,
-      incident_id: None = None,
-      sketch_id: Optional[int] = 0,
-      analyzers: None = None,
-      token_password: str = '',
+      incident_id: str,
+      sketch_id: Optional[int],
+      analyzers: str,
+      token_password: str,
+      endpoint: Optional[str],
```

Comment:
Optional seems weird, since you will never be able to connect without this being set somehow?

Reply:
It's optional because `token_password` implies that you already have the endpoint and everything stored in the token.